react/OrderSeller.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { useRuntime } from 'vtex.render-runtime'
+
+import { useOrder } from './components/OrderContext'
+
+const OrderSeller: FC = () => {
+  const runtime = useRuntime()
+  const order = useOrder()
+
+  const storeAccount = runtime.account
+  const orderSeller = order.sellers[0].name
```

Comment:
same case, assuming `sellers` is `undefined` what happens to `name`?

Reply:
You can check the `Order` type here. `sellers` cannot be `undefined`.